src/content/pages/faq.mdx

```diff
@@ -60,7 +59,7 @@ Share your ideas with the programme team at [programme@europython.eu](mailto:pro
 
 ## **Q. When can I submit a talk proposal?**
 
-A. The [Call for Proposals (CfP)](/programme/cfp) is open from **10 – <strike>27 January</strike> 3rd February 2025** (extended).
+A. Sadly, at this point you cannot. The [Call for Proposals (CfP)](/programme/cfp) was open from **10 – <strike>27 January</strike> 3rd February 2025** (extended).
```

Comment:
Same question as above. Should we keep the extension mentioned?

Reply:
I think it makes sense to keep info about the fact that the CFP was extended, for the sake of record-keeping.

Reply:
ok.